src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -33,13 +33,14 @@
 {defects}
 
 You can run this analysis locally with:
+
```

Comment:
Nit: probably unnecessary? We don't have it before {defects} and the output is the same

Reply:
Yeap, i wanted to test the output on Phabricator as it's not standard markdown